### ISSUE TITLE
Use text/template for Syslog.Output.Template

### ISSUE THREAD

**Issue by dsvensson** [webdevops/go-syslogd#1]
Would you accept a patch to utilize `text/template` instead of simply `%s` formatting for the Template configuration directive?

**Comment by mblaschke** [comment]
Sure :)
Would it make sense to detect if `%s` is configured to use Sprintf-Style and otherwise use `text/template`?

**Comment by dsvensson** [comment]
Yeah, not that pretty but that was what I was thinking about as well.
